x-pack/solutions/observability/plugins/observability_onboarding/public/application/quickstart_flows/otel_apm/index.tsx

```diff
@@ -0,0 +1,308 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+import type { EuiBasicTableColumn } from '@elastic/eui';
+import {
+  EuiBasicTable,
+  EuiButtonIcon,
+  EuiCallOut,
+  EuiLink,
+  EuiMarkdownFormat,
+  EuiPanel,
+  EuiSkeletonText,
+  EuiSpacer,
+  EuiSteps,
+  EuiText,
+  copyToClipboard,
+  useEuiTheme,
+} from '@elastic/eui';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import type { ValuesType } from 'utility-types';
+import { usePerformanceContext } from '@kbn/ebt-tools';
+import { FETCH_STATUS } from '@kbn/observability-shared-plugin/public';
+import type { ObservabilityOnboardingAppServices } from '../../..';
+import { useFlowBreadcrumb } from '../../shared/use_flow_breadcrumbs';
+import { FeedbackButtons } from '../shared/feedback_buttons';
+import { GetStartedPanel } from '../shared/get_started_panel';
+import { useOtelApmFlow } from './use_otel_apm_flow';
+import { EmptyPrompt } from '../shared/empty_prompt';
+
+export function OtelApmQuickstartFlow() {
+  useFlowBreadcrumb({
+    text: i18n.translate('xpack.observability_onboarding.otelApm.breadcrumbs.k8sOtel', {
+      defaultMessage: 'Application: OpenTelemetry',
+    }),
+  });
+  const {
+    services: {
+      share,
+      context: { isServerless },
+    },
+  } = useKibana<ObservabilityOnboardingAppServices>();
+  const { data, status, error, refetch } = useOtelApmFlow();
+  const { onPageReady } = usePerformanceContext();
+
+  useEffect(() => {
+    if (data) {
+      onPageReady({
+        meta: {
+          description: `[ttfmp_onboarding] Request to create the onboarding flow succeeded and the flow's UI has rendered`,
+        },
+      });
+    }
+  }, [data, onPageReady]);
+
+  if (error !== undefined) {
+    return <EmptyPrompt onboardingFlowType="otel_apm" error={error} onRetryClick={refetch} />;
+  }
+
+  const apmLocator = share.url.locators.get('APM_LOCATOR');
+
+  return (
+    <EuiPanel hasBorder paddingSize="xl">
+      {!isServerless && (
+        <EuiCallOut
+          announceOnMount
+          title={i18n.translate(
+            'xpack.observability_onboarding.otelApmQuickstartFlow.euiCallOut.calloutTitleLabel',
+            {
+              defaultMessage:
+                'This is a Tech Preview feature and should not be used in production.',
+            }
+          )}
+          iconType="info"
+          color="warning"
+        >
+          <p>
+            {i18n.translate(
+              'xpack.observability_onboarding.otelApmQuickstartFlow.p.descriptionTextGoesHereLabel',
+              {
+                defaultMessage:
+                  'For more details on how data is stored and how it compares to previous OpenTelemetry functionality, refer to Comparison between mOTLP and APM Server.',
```

Comment:
this looks like this should link to some docs page but it doesn't, is this an oversight?